HW15/logger.py

```diff
@@ -0,0 +1,4 @@
+import logging
+
+def setup_logger(logfile):
+    logging.basicConfig(filename=logfile, level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
```

Comment:
Я бы делал возврат объекта логгера:
```suggestion
import logging

def setup_logger(logfile):
    logging.basicConfig(filename=logfile, level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
    return logging.getLogger()
```

Reply:
Так и нужно, а еще лучше добить до декоратора